src/pages/admin/AdminNavbar.js

```diff
@@ -1,86 +1,86 @@
+          <span onClick={signOut}>Logout</span>
+        </div>
+      </Popover.Content>
+    </Popover>
+  );
 
-    return (
-        <nav className="navbar navbar-light bg-light navbar-expand-sm rounded fixed-top position-sticky">
-            <a
-                className="navbar-brand ml-sm-5 px-3"
-                href="/admin"
-                target="_self"
-                rel="noopener noreferrer"
-            >
-                <img src={jjcIcon} alt="JJC logo" />
-            </a>
-            <button
-                className="navbar-toggler"
-                type="button"
-                data-toggle="collapse"
-                data-target="#myTogglerNav"
-                aria-controls="myTogglerNav"
-                aria-expanded="false"
-                aria-label="Toggle navigation"
-            >
-                <span className="navbar-toggler-icon" />
-            </button>
+  return (
+    <nav className="navbar navbar-light bg-light navbar-expand-sm rounded fixed-top position-sticky">
+      <a
+        className="navbar-brand ml-sm-5 px-3"
+        href="/admin"
+        target="_self"
+        rel="noopener noreferrer"
+      >
+        <img src={jjcIcon} alt="JJC logo" />
+      </a>
+      <button
+        className="navbar-toggler"
+        type="button"
+        data-toggle="collapse"
+        data-target="#myTogglerNav"
+        aria-controls="myTogglerNav"
+        aria-expanded="false"
+        aria-label="Toggle navigation"
+      >
+        <span className="navbar-toggler-icon" />
+      </button>
 
-            <div className="collapse navbar-collapse" id="myTogglerNav">
-                <div className="left-section">
-                    <div>
-                        <BsBell size="1.5em" color="#606060"/>
-                        <sup className="notification-number"><Badge pill variant="danger">10</Badge></sup>
-                    </div>
-                    <div className="vertical-line mr-3 ml-3"></div>
-                    <Avatar />
-                    <h6 className="m-3">
-                        Hi,
-                        {auth()?.user === undefined ? 'Dude' : auth().user}
-                    </h6>
-                    <OverlayTrigger trigger="click" placement="bottom" overlay={userDropdown} rootClose>
-                        <img src={UserDropDownIcon} alt="UserDropdownIcon" className="mr-sm-5 user-profile-dropdown-icon" />
-                    </OverlayTrigger>
-                </div>
-            </div>
-        </nav>
-    );
+      <div className="collapse navbar-collapse" id="myTogglerNav">
+        <div className="left-section">
+          <div>
+            <BsBell size="1.5em" color="#606060" />
+            <sup className="notification-number"><Badge pill variant="danger">10</Badge></sup>
+          </div>
+          <div className="vertical-line mr-3 ml-3" />
+          <Avatar imgSrc={Object.keys(globalState.profile).length > 0 ? globalState.profile.profilePicture : ''} />
```

Comment:
Instead of repeatedly accessing globalState.profile and verifying if it exists (by looping over the keys), how about we store it into a variable after accessing globalState ? Then we can directly say profile.profilePicture || ''